grunt/config/translate.js

```diff
@@ -1,10 +1,10 @@
 module.exports = function(grunt) {
   return {
-    masterLang: "en",
+    masterLang: 'en',
     targetLang: null,
-    format: "csv",
-    csvDelimiter: ",",
-    "shouldReplaceExisting": false
+    format: 'csv',
+    csvDelimiter: ',',
+    'shouldReplaceExisting': false
```

Comment:
Minor: Is the quote wrapping necessary for this 'shouldReplaceExisting'?

Reply:
It isn't, but it was just the linter's auto-fix which replaced the previous quote marks in this instance.